int32 g_vehicle_num
string[] uri_v
string[] tf_prefix_v
float32[] roll_trim_v
float32[] pitch_trim_v
bool enable_logging
bool enable_parameters
LogBlock[] log_blocks
bool use_ros_time
bool enable_logging_imu
bool enable_logging_temperature
bool enable_logging_magnetic_field
bool enable_logging_pressure
bool enable_logging_battery
---
